src/app/components/ErrorMain/__snapshots__/index.test.jsx.snap

```diff
@@ -287,8 +289,7 @@ exports[`ErrorMain should correctly render for an error page for News 1`] = `
       </span>
       <h1
         class="emotion-7 emotion-8"
-        id="content"
-        tabindex="-1"
+        id="headline"
```

Comment:
Why is this now called headline? What is this id for?

Reply:
Hi @greenc05, since we are using the id of `content` on the wrapper instead of the `h1` we needed a replacement `id` and `header` worked perfectly

Reply:
Thanks, and what is this id used for?

Reply:
No, we are not using it anywhere else